tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -334,15 +352,13 @@
           .labelFormatter(function(n){ return Number(n).toFixed(2) + "%" ;});
         var plot = new Plottable.Components.Table([[pie, legend]]);
         plot.renderTo(div);
-
       }
     },
 
     /* Draws a table of Dataset Op statistics */
     _showHostTable: function(hostJson) {
-      var inputOpsTable = document.getElementById('host-side-table');
-      var inputOpsTableBody = document.createElement('tbody');
-      Polymer.dom(inputOpsTable).appendChild(inputOpsTableBody);
+      var inputOpsTableBody = this.$.host_side_table;
```

Comment:
Oh, that's why this ID got moved to the `<tbody>` element. Could we change the ID name to `host_side_table_content`?

Reply:
done.